packages/core/src/extension.ts

```diff
@@ -171,22 +171,32 @@ export async function activate(context: vscode.ExtensionContext) {
                 switchToAmazonQCommand.register()
                 installAmazonQExtension.register()
 
-                if (!isExtensionInstalled(VSCODE_EXTENSION_ID.amazonq) && isPreviousQUser()) {
-                    void vscode.window
-                        .showInformationMessage(
-                            'Amazon Q has moved to its own VSCode extension.' +
-                                '\nInstall to work with Amazon Q, a generative AI assistant, with chat and code suggestions.',
-                            'Install',
-                            'Learn More',
-                            'Dismiss'
-                        )
-                        .then(async resp => {
-                            if (resp === 'Learn More') {
-                                void learnMoreAmazonQCommand.execute()
-                            } else if (resp === 'Install') {
-                                void installAmazonQExtension.execute()
-                            }
-                        })
+                if (!isExtensionInstalled(VSCODE_EXTENSION_ID.amazonq)) {
+                    if (isPreviousQUser()) {
+                        await installAmazonQExtension.execute()
+                        void vscode.window
+                            .showInformationMessage('We have installed Amazon Q extension for you', 'Restart')
+                            .then(async resp => {
+                                if (resp === 'Restart') {
```

Comment:
If we are only reloading the window, then let's use the same language that vscode does.

<img width="155" alt="image" src="https://github.com/aws/aws-toolkit-vscode/assets/55561878/ce84af41-4235-43fe-a3f0-678fbe900dcb">

```suggestion
                                if (resp === 'Reload Now') {
```